easybuild/easyconfigs/o/OR-Tools/OR-Tools-7.1-fix-makefiles.patch

```diff
@@ -0,0 +1,81 @@
+# Use EB deps and install prefix
+# wpoely86@gmail.com
+diff -Nur or-tools-7.1.orig/Makefile.local.eb or-tools-7.1/Makefile.local.eb
+--- or-tools-7.1.orig/Makefile.local.eb	1970-01-01 01:00:00.000000000 +0100
++++ or-tools-7.1/Makefile.local.eb	2019-05-20 20:49:11.941360942 +0200
+@@ -0,0 +1,5 @@
++UNIX_SWIG_BINARY = $(EBROOTSWIG)/bin/swig
++JAVA_HOME = $(EBROOTJAVA)
++UNIX_PYTHON_VER = 3.7
++UNIX_GUROBI_DIR=$(EBROOTGUROBI)
++GUROBI_LIB_VERSION=$(shell echo $(EBVERSIONGUROBI) | sed 's:\([0-9]\+\)\.\([0-9]\+\)\..*:\1\2:')
+diff --git a/Makefile b/Makefile
+index c70b93c..c1d5a63 100755
+--- a/Makefile
++++ b/Makefile
+@@ -55,6 +55,7 @@ OR_ROOT_FULL=$(OR_TOOLS_TOP)
+ 
+ # Load local variables
+ include $(OR_ROOT)Makefile.local
++include $(OR_ROOT)Makefile.local.eb
+ 
+ # Change the file extensions to increase diff tool friendliness.
+ # Then include specific system commands and definitions
+diff --git a/makefiles/Makefile.python.mk b/makefiles/Makefile.python.mk
+index 2b548e0..b7f72ae 100755
+--- a/makefiles/Makefile.python.mk
++++ b/makefiles/Makefile.python.mk
+@@ -1113,7 +1113,8 @@ endif # ifneq ($(PYTHON_EXECUTABLE),)
+ 
+ .PHONY: install_python # Install Python OR-Tools on the host system
+ install_python: pypi_archive
+-	cd "$(PYPI_ARCHIVE_TEMP_DIR)$Sortools" && "$(PYTHON_EXECUTABLE)" setup.py install --user
++	mkdir -p $(DESTDIR)/lib/python3.7/site-packages
```

Comment:
@wpoely86 Can we avoid hardcoding the `3.7` here?

Maybe use `$(UNIX_PYTHON_VER)` here, and define `UNIX_PYTHON_VER` using `$EBVERSIONPYTHON`?

Reply:
`3.7` is still hardcoded?

Reply:
oeps, forgot to change. fixed now